gulpfile.js

```diff
@@ -41,15 +41,20 @@ const jsPolyfills = './assets/js/libs/polyfills.js';
 const jsSource = ['./assets/js/**/*.js', '!' + js3rdPartySource, '!' + jsPolyfills];
 const jsDest = './source/assets/js';
 
-let options = minimist(process.argv);
-let environment = options.environment || 'development';
-let mocha_grep = options['mocha-grep'] || null;
+let options = minimist(process.argv, {'boolean': ['sass-lint'], 'default': {'sass-lint': true, 'environment': 'development', 'mocha-grep': null}});
+let environment = options.environment;
+let mochaGrep = options['mocha-grep'];
+let sassLint = options['sass-lint'];
```

Comment:
this can probably be made more general, but depends on what you want `gulp` commands to look like

Reply:
This works. Confirmed `sass-lint` skipped whilst generating non-minified output through running `./node_modules/.bin/gulp default --sass-lint=false`